schema/NATIONAL_PARTY_PARTNERSHIP_RECEIPTS.json

```diff
@@ -392,7 +392,6 @@
             "type": "string",
             "minLength": 1,
             "maxLength": 100,
-            "pattern": "^([ -~]{0,39} \\(Partnership attributions do not meet itemization threshold\\)|[ -~]{0,61} \\(See Partnership Attribution\\(s\\) below\\))$",
```

Comment:
This one doesn't need to change because the variable is not the contact but rather "Pres. Nominating Convention Account" / "Headquarters Buildings Account" / "Recount/Legal Proceedings Account

Reply:
Removed!